e2e/bitrise.yml

```diff
@@ -0,0 +1,260 @@
+  #    after_run:
+  #      - _common
+
+  #  test_swiftpm:
+  #    envs:
+  #      - XCODEBUILD_OPTIONS:
+  #      - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-ios-swiftpm.git
+  #      - BRANCH: master
+  #      - BITRISE_PROJECT_PATH: sample-swiftpm.xcodeproj
+  #      - BITRISE_SCHEME: "sample swiftpm"
+  #      - OUTPUT_TOOL: xcodebuild
+  #      - SIMULATOR_OS_VERSION: "latest"
+  #      - OUTPUT_DIR: $BITRISE_DEPLOY_DIR
+  #      - BITRISE_APP_DIR_PATH_EXPECTED: $BITRISE_DEPLOY_DIR/sample-swiftpm.app
+  #      - BITRISE_APP_DIR_PATH_LIST_EXPECTED: $BITRISE_DEPLOY_DIR/sample-swiftpm.app
+  #      - CODE_SIGNING_ALLOWED: "no"
+  #    after_run:
+  #      - _common
+
+  _common:
+    steps:
+      - script:
+          inputs:
+            - content: |-
+                #!/bin/bash
+                set -ex
+                rm -rf "$ORIG_BITRISE_SOURCE_DIR/_tmp"
+                mkdir -p "$ORIG_BITRISE_SOURCE_DIR/_tmp"
+      - change-workdir:
+          title: Switch working dir to _tmp dir
+          run_if: "true"
+          inputs:
+            - path: "$ORIG_BITRISE_SOURCE_DIR/_tmp"
+            - is_create_path: true
+      - script:
+          inputs:
+            - content: |-
+                #!/usr/bin/env bash
+
+                if [[ -z "${SAMPLE_APP_URL}" ]]; then
+                  echo "error: there is no SAMPLE_APP_URL env var specified"
+                  exit 1
+                elif [[ -z "${COMMIT}" && -z "${BRANCH}" ]]; then
+                  echo "error: can't checkout: there is no BRANCH or COMMIT env var specified"
+                  exit 1
+                fi
+
+                git init
+                git remote add origin "${SAMPLE_APP_URL}"
+                git fetch || exit 1
+                [[ -n "${COMMIT}" ]] && git checkout "${COMMIT}" || git checkout "${BRANCH}"
+
+      - script:
+          title: Install yarn deps if needed
+          inputs:
+            - content: |-
+                #!/usr/bin/env bash
+                set -e
+
+                if [ -f yarn.lock ]; then
+                  echo "yarn.lock file found in the root dir. Running yarn install."
+                  yarn install
+                else
+                  echo "No yarn.lock file found in the root dir."
+                fi
+      - script:
+          title: Install cocoapods if needed
+          inputs:
+            - content: |-
+                #!/usr/bin/env bash
+                set -e
+
+                if [ -f Podfile.lock ]; then
+                  echo "Podfile.lock file found in the root dir. Running pod install."
+                  pod install
+                elif [ -f ios/Podfile.lock ]; then
+                  echo "Podfile.lock file found in the /ios dir. Running pod install."
+                  cd ios
+                  pod install
+                  cd ..
+                else
+                  echo "No Podfile.lock file found."
+                fi
+      - path::./:
```

Comment:
nit: missing `title` attribute. I like giving a title to these steps (`Execute step`, for example) because it makes the build logs more readable.

Reply:
- [ ] Add `title` to `e2e/bitrise.yml` step